src/Symfony/Component/DependencyInjection/Tests/Fixtures/yaml/services9.yml

```diff
@@ -76,6 +76,10 @@ services:
         class: stdClass
         decorates: decorated
         decoration_inner_name: decorated.pif-pouf
+    deprecated_service:
+        class: stdClass
+        deprecated: true
```

Comment:
But then, for YAML and XML, I could merge the two attributes (allowing a `null` value to use the default message), and just the presence of `deprecated` be enough to mark the service as deprecated...

But then again, we wouldn't be able to override the deprecation status in a inherited / decorated service, as it is currently the case in XML